{"success": true, "data": [{"type": "correct", "user": {"id": 457, "name": "3v1L@D3V1L"}, "challenge_id": 54, "date": "2024-04-26T14:19:38.630332+00:00", "id": 759, "challenge": {"id": 54, "name": "sanitycheck", "value": 1, "category": "misc"}, "team": {"id": 72, "name": "PrivateUser"}}], "meta": {"count": 1}}
